From cd7802fffbc7140c299fb28ac613041794e662a4 Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Tue, 3 May 2011 13:12:40 -0500
Subject: ENGR00142950-5 - iMX5 defconfig: Port EPDC/PxP/MAX17135 to 2.6.38

- Added support for EPDC driver
- Added support for PxP driver
- Added support for MAX17135 EPD PMIC driver

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 arch/arm/configs/imx5_defconfig |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/configs/imx5_defconfig b/arch/arm/configs/imx5_defconfig
index d9f524f..df5ddbd 100755
--- a/arch/arm/configs/imx5_defconfig
+++ b/arch/arm/configs/imx5_defconfig
@@ -276,6 +276,8 @@ CONFIG_IMX_HAVE_PLATFORM_IMX_DCP=y
 CONFIG_IMX_HAVE_PLATFORM_RANDOM_RNGC=y
 CONFIG_IMX_HAVE_PLATFORM_PERFMON=y
 CONFIG_IMX_HAVE_PLATFORM_LDB=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_PXP=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_EPDC=y
 
 #
 # Freescale MXC Implementations
@@ -1158,6 +1160,7 @@ CONFIG_HWMON=y
 # CONFIG_SENSORS_LM95241 is not set
 # CONFIG_SENSORS_MAX1111 is not set
 # CONFIG_SENSORS_MAX1619 is not set
+CONFIG_SENSORS_MAX17135=y
 # CONFIG_SENSORS_MAX6650 is not set
 # CONFIG_SENSORS_PC87360 is not set
 # CONFIG_SENSORS_PC87427 is not set
@@ -1239,6 +1242,7 @@ CONFIG_MFD_MC13XXX=y
 # CONFIG_EZX_PCAP is not set
 # CONFIG_MFD_TPS6586X is not set
 # CONFIG_MFD_WL1273_CORE is not set
+CONFIG_MFD_MAX17135=y
 CONFIG_REGULATOR=y
 # CONFIG_REGULATOR_DEBUG is not set
 # CONFIG_REGULATOR_DUMMY is not set
@@ -1261,6 +1265,7 @@ CONFIG_REGULATOR_MC13892=y
 # CONFIG_REGULATOR_AD5398 is not set
 # CONFIG_REGULATOR_TPS6524X is not set
 CONFIG_REGULATOR_DA9052=y
+CONFIG_REGULATOR_MAX17135=y
 CONFIG_MEDIA_SUPPORT=y
 
 #
@@ -1500,6 +1505,9 @@ CONFIG_FB_MODE_HELPERS=y
 #
 # Frame buffer hardware drivers
 #
+CONFIG_FB_MXC=y
+CONFIG_FB_MXC_EINK_PANEL=y
+# CONFIG_FB_MXC_EINK_AUTO_UPDATE_MODE is not set
 # CONFIG_FB_UVESA is not set
 # CONFIG_FB_S1D13XXX is not set
 # CONFIG_FB_TMIO is not set
@@ -1930,6 +1938,8 @@ CONFIG_DMADEVICES=y
 #
 # DMA Devices
 #
+CONFIG_MXC_PXP=y
+CONFIG_MXC_PXP_CLIENT_DEVICE=y
 # CONFIG_TIMB_DMA is not set
 CONFIG_IMX_SDMA=y
 CONFIG_DMA_ENGINE=y
-- 
1.7.9.5

